### PR TITLE
Reject empty term

### PR DESCRIPTION
**Pull request by pchampin**
***

### :boom: Error: write EPROTO 140222018242432:error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version:../deps/openssl/openssl/ssl/s23_clnt.c:772:
 :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Mar 16, 2020, 11:38 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [HTML Diff Service](http://services.w3.org/htmldiff) - The HTML Diff Service is used to create HTML diffs of the spec changes suggested in a pull request.

:link: [Related URL](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fpr-preview.s3.amazonaws.com%2Fw3c%2Fjson-ld-api%2Fpull%2F421%2F257c09e.html&doc2=https%3A%2F%2Fpr-preview.s3.amazonaws.com%2Fw3c%2Fjson-ld-api%2Fpull%2F421.html)

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/json-ld-api%23421.)._
</details>

### CODE REVIEW

**Comment by pchampin** [comment]
This PR addresses #379

**Comment by davidlehn** [comment]
Need to add the same toRdf test.  Also "e052" will bump into the recursion test PR naming.  May be easier to handle that here with less tests to rename?

Sad to see such a nice obfuscated json-ld feature get testsuited away.

**Comment by gkellogg** [comment]
> Need to add the same toRdf test. Also "e052" will bump into the recursion test PR naming. May be easier to handle that here with less tests to rename?

Yes, we should compare with other PRs and rename as appropriate, and add a toRdf version. (It would be nice to have a way to not have to do this manually!)

> Sad to see such a nice obfuscated json-ld feature get testsuited away.

👍 But, that's what the spec says to do; perhaps 2.0 can roll that back.

File: tests/expand-manifest.html [hunk 1]
```diff
@@ -5353,6 +5353,27 @@ <h2>
 </dd>
 </dl>
 </dd>
+<dt id='te052'>
```
Comment (davidlehn) [inline tests/expand-manifest.html]: Missed an html rebuild.

**Comment by gkellogg** [comment]
I'm rebuilding them all as part of PR #416.

**Pull request merged**